<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>284</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="levelBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>351</width>
      <height>71</height>
     </rect>
    </property>
    <property name="title">
     <string>level</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>331</width>
       <height>31</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="level1">
        <property name="text">
         <string>1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level2">
        <property name="text">
         <string>2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level3">
        <property name="text">
         <string>3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level4">
        <property name="text">
         <string>4</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level5">
        <property name="text">
         <string>5</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level6">
        <property name="text">
         <string>6</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="level7">
        <property name="text">
         <string>7</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="levelAuto">
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">levelGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QLabel" name="labelLevelShow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>LevelShow</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdLevelNumber">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="smallDecimalPoint">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="funcBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Function</string>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>71</width>
       <height>146</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="levelChoseButton">
        <property name="text">
         <string>Level</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">funcGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ADC0">
        <property name="text">
         <string>ADC0</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">funcGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ADC1">
        <property name="text">
         <string>ADC1</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">funcGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ADC2">
        <property name="text">
         <string>ADC2</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">funcGroup</string>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ADC3">
        <property name="text">
         <string>ADC3</string>
        </property>
        <attribute name="buttonGroup">
         <string notr="true">funcGroup</string>
        </attribute>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ADCshow">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>89</y>
      <width>151</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>ADCshow</string>
    </property>
    <widget class="QTextBrowser" name="ADCtext">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>131</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AuthorBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="title">
     <string>Author</string>
    </property>
    <widget class="QGraphicsView" name="Author">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>161</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/new/prefix1/MZ.jpg);</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="AuthorInfo">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>161</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border:0;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>33</x>
       <y>150</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 11pt &quot;Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>By : AaronYin - MZ</string>
     </property>
     <property name="textFormat">
      <enum>Qt::AutoText</enum>
     </property>
    </widget>
    <zorder>AuthorInfo</zorder>
    <zorder>label</zorder>
    <zorder>Author</zorder>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="funcGroup"/>
  <buttongroup name="levelGroup"/>
 </buttongroups>
</ui>
